File: Java/HCF.java
package com.lev;

import java.util.Scanner;

public class HCF {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the numbers : ");
        int n1 = input.nextInt();
        int n2 = input.nextInt();
        int val;
        if(n1 > n2){
            val = n1;
        }else{
            val= n2;
        }
        while (true){
            if((val % n1 == 0) && (val % n2 == 0)){
                break;
            }else{
                val++;
            }
        }
        System.out.println("LCM : "+val);
    }
}

File: Python/HCF.py
print("Enter two numbers: ",end='')
n1=int(input())
n2=int(input())
if(n1>n2):
    val=n1
else:
    val=n2
while(True):
    if((n1%val==0) and (n2%val==0)):
        break
    else:
        val=val-1
print("HCF of",n1,"and",n2,":",val)
